usr/sbin/truenas-install

```diff
@@ -287,11 +287,15 @@ create_partitions()
             return 1
     fi
     # Create EFI partition (Even if not used, allows user to switch to UEFI later)
-    if ! sgdisk -n2:1M:+512M -t2:EF00 /dev/${_disk}; then
+    if ! sgdisk -n2:1024K:+524288K -t2:EF00 /dev/${_disk}; then
+	    return 1
+    fi
+    # Create swap partition
+    if ! sgdisk -n3:525312K:+1048576K -t3:8200 /dev/${_disk}; then
```

Comment:
I think we are supposed to create swap partition at the end, and only if the disk is large enough? Isnt that what we do in freebsd installer?